dialogflow-add.js

```diff
@@ -1,32 +1,21 @@
-//This is the upload file of google.
-//It first read the data file and transfer it to the upload file.
-//Because of lacking the permission to certain module of nodejs,
-//invoke the python file to finish it.
-
-
 'use strict';
 function createIntents() {
-    const grpc = require('grpc');
-    const prompt = require('prompt');
     this.uploadPhase =function(Intent,projectId){
        const dialogflow = require('dialogflow');
-       const contextsClient = new dialogflow.ContextsClient();
        const intentsClient = new dialogflow.IntentsClient();
        const agentPath = intentsClient.projectAgentPath(projectId);
     const Request = {
         parent: agentPath,
-        intent: Intent,
+        intent: Intent
     };
     intentsClient
         .createIntent(Request)
         .then(responses => {
         console.log("Intent has been created:");
```

Comment:
When you get some response, better to return this response back, instead of doing a `console.log`.

You can design your function to have a callback function, or use async syntax to return the results. In this way, you could return back error status when you encountered some errors, then the outside function will be able to deal with that.